app/assets/stylesheets/mixins.scss

```diff
@@ -53,3 +53,18 @@
      color: darken($color, 20%);
    }
 }
+
+@mixin btn-main($color){
```

Comment:
I am wondering if it would not be better to name `card-base` and `button-base` ?